dashboard/modules/dashboard_sdk.py

```diff
@@ -221,6 +224,20 @@ def __init__(
         # Headers used for all requests sent to job server, optional and only
         # needed for cases like authentication to remote cluster.
         self._headers = cluster_info.headers
+        # Set SSL verify parameter for the requests library and create an ssl_context
+        # object when needed for the aiohttp library.
+        self._verify = verify
+        if isinstance(self._verify, str):
+            if os.path.isdir(self._verify):
+                cafile, capath = None, self._verify
+            else:
+                cafile, capath = self._verify, None
```

Comment:
```suggestion
                cafile, capath = self._verify, None
```
Should we also check if it's an existing file and raise a ValueError with a friendly message if it's neither an existing file nor existing dir?

Reply:
Makes sense! I had delegated that to the underlying libraries but no harm in adding that in here.